applications/intro/site-local/group.yml

```
@@ -0,0 +1,6 @@
+documentation_complete: true
+
+title: "Site Specific Information"
+
+description: |-
+  {{{ xccdf_value("site_local_message") }}}
```

```suggestion
  {{{ xccdf_value("var_site_local_message") }}}
```

This could help - the `.var` suffix is not part of the variable name, but the `var_` prefix is, as it has no special meaning.